src/main/java/com/gregtechceu/gtceu/api/item/PipeBlockItem.java

```diff
@@ -47,10 +47,18 @@ public PipeBlock getBlock() {
     public void appendHoverText(ItemStack stack, @Nullable Level level, List<Component> tooltip,
                                 TooltipFlag isAdvanced) {
         super.appendHoverText(stack, level, tooltip, isAdvanced);
-        if (GTUtil.isShiftDown()) {
-            tooltip.add(Component.translatable("gtceu.tool_action.wire_cutter.connect"));
+        if (this instanceof OpticalPipeBlockItem) {
+            if (GTUtil.isShiftDown()) {
+                tooltip.add(Component.translatable("gtceu.tool_action.wire_cutter.connect"));
+            } else {
+                tooltip.add(Component.translatable("gtceu.tool_action.show_tooltips"));
+            }
```

Comment:
just override it in OpticalPipeBlockItem?

Reply:
If you override it, then you have to call super().
But if you call super() it'll add the wrench tooltip.
But if you don't call super then you don't get the block tooltip unless you manually call `this.getBlock().appendHoverText()` in the override for this tooltip.
![image](https://github.com/user-attachments/assets/8bb55fd0-9185-4633-9e51-70c19023606d)
It's a mess overall D: